lib/media/drm_engine.js

```diff
@@ -698,7 +698,7 @@ shaka.media.DrmEngine = class {
     // Create a config entry for each key system.
     for (const info of allDrmInfo) {
       const config = {
-        // Ignore initDataTypes.
+        initDataTypes: ['cenc'],
```

Comment:
ℹ️  Like #2750, initDataTypes is needed for the requestMediaKeySystemAccess to work properly